eng/common/templates/jobs/publish.yml

```
@@ -134,6 +143,33 @@ jobs:
         --architecture '*'
         $(dryRunArg)
         $(imageBuilder.commonCmdArgs)
+  - template: /eng/common/templates/steps/run-imagebuilder.yml@self
+    parameters:
+      displayName: Generate EOL Annotation Data
+      serviceConnection: $(publish.serviceConnectionName)
+      internalProjectName: ${{ parameters.internalProjectName }}
```

```suggestion
      internalProjectName: internal
```

Feel free to disregard, but you could consider hard-coding these when adding new pipeline code. Related: https://github.com/dotnet/docker-tools/issues/1267